DEPS

```diff
@@ -537,10 +537,11 @@ deps = {
     'dep_type': 'cipd',
     'condition': 'host_os == "mac"',
   },
-  'src/buildtools/third_party/libc++/trunk':
-    Var('chromium_git') +
-    '/external/github.com/llvm/llvm-project/libcxx.git' + '@' +
-    Var('libcxx_revision'),
+  # Cobalt: imported
+#   'src/buildtools/third_party/libc++/trunk':
+#     Var('chromium_git') +
+#     '/external/github.com/llvm/llvm-project/libcxx.git' + '@' +
+#     Var('libcxx_revision'),
```

Comment:
… and 2 spaces after the `#` on lines 541–544,
consistent with for example lines 886–888 below.